### DISCUSSION TITLE
Welcome to yolov8-object-tracking Discussions!

### DISCUSSION THREAD

**Discussion by RizwanMunawar** [RizwanMunawar/yolov8-object-tracking#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rajkumar-1823** [comment]
Hi i have followed your step but there is a error showing for me is ""Python was not found; run without arguments to install from the Microsoft Store, or disable this shortcut from Settings"" else it shows ""FileNotFoundError: [WinError 2] The system cannot find the file specified"" Can you please clear it.

**Comment by RizwanMunawar** [comment]
@Mr-Single18 you will need to install python on your system from the mentioned link below.
https://www.python.org/ftp/python/3.9.11/python-3.9.11-amd64.exe

**Comment by 97mallikan** [comment]
Hi,
I am using this git code for one of my research work. For that I need to site this.
I need the information for citation.
